app.js

```diff
@@ -0,0 +1,41 @@
+require("dotenv").config();
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const { DataSource } = require("typeorm");
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("combined"));
+
+const database = new DataSource({
+  type: process.env.DB_TYPE,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+});
+
+app.get("/ping", function (req, res, next) {
+  res.json({ message: "pong!" });
+});
```

Comment:
- 화살표 함수로 수정해주세요.
- next 함수는 사용하지 않으니 삭제해주세요.
- return 키워드를 통해 함수가 종료되는 코드 라인을 명확히 구분해주세요!

```suggestion
app.get("/ping", (req, res) => {
  return res.json({ message: "pong!" });
});
```